README.md

```diff
@@ -82,4 +82,3 @@ run the pipeline in github
 or 
 docker build --target test --tag todo-app:test .
 docker run --env-file .env.test todo-app:test
-
```

Comment:
It would be worthwhile adding the link to your Azure website here since otherwise there's no way to know where that is!